### PR TITLE
Modified Hover Button Behaviour

### PR DESCRIPTION
**Pull request by Balaguru4580**
White button now exhibits hover button behaviour. 
Red and Yellow hover buttons tweaked for clarity.

Issue:

### CODE REVIEW

File: src/sbuttons.less [hunk 1]
```diff
@@ -1,16 +1,16 @@
 @blue: #50bfe6;
 @darkBlue: #1fa5d6;
 @pink: #d53caf;
-@darkPink: #be309a;
+@darkPink: #c4309f;
```
Comment (shahednasser) [inline src/sbuttons.less:4]: change this to `#c10c94`

File: src/sbuttons.less [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 @darkerBlue: #3193b5;
 @lighterBlue: #19c3ff;
 @red: #ed0a3f;
-@darkRed: #bf0833;
+@darkRed: #cf0837;
 @green: #3aa655;
 @darkGreen: #328f4a;
 @yellow: #fbe870;
-@darkYellow: #f9d90b;
+@darkYellow: #f3da37;
```
Comment (shahednasser) [inline src/sbuttons.less:13]: change to `#f7da22`

**Review by shahednasser — CHANGES_REQUESTED** [review]

File: src/sbuttons.less (hunk 2 above)
Comment (Balaguru4580) [reply src/sbuttons.less]: Alright, the colors have been changed.

**Comment by shahednasser** [comment]
Ok great work thank you.

@all-contributors please add @Balaguru4580 for code

**Comment by Balaguru4580** [comment]
Thanks!

**Pull request merged**